tests/test_material.py

```diff
@@ -44,6 +76,9 @@ def test_metal_voxel(self):
     def test_plastic(self):
         self.assertTrue(material_spawn_test(pi.Plastic))
 
+    def test_paint(self):
+        self.assertTrue(paint_position_test(pi.Paint))
```

Comment:
It makes no scense to use a single use function, instead directly write the function as part of test_paint

Reply:
You should also run the material spawn test